runtimes/cuda/cuda.zig

```diff
@@ -12,6 +17,23 @@ fn hasNvidiaDevice() bool {
     return true;
 }
 
+fn setupXlaGpuCudaDirFlag() !void {
```

Comment:
I find code easier to read when this kind of internal helper function is located at the bottom of the file,
or at least after their call site